go/vt/vttablet/tabletmanager/vreplication/vdbclient.go

```diff
@@ -92,8 +92,8 @@ func (vc *vdbClient) ExecuteFetch(query string, maxrows int) (*sqltypes.Result,
 
 // Execute is ExecuteFetch without the maxrows.
 func (vc *vdbClient) Execute(query string) (*sqltypes.Result, error) {
-	// Number of rows should never exceed relayLogMaxSize.
-	return vc.ExecuteFetch(query, relayLogMaxSize)
+	// Number of rows should never exceed relayLogMaxItems.
+	return vc.ExecuteFetch(query, *relayLogMaxItems)
```

Comment:
Before this PR, the allowed number of rows was 30K. With the changes in this PR, that defaults to 1K (though the new flag let us increase the limit). @rohit-nayak-ps is this an acceptable change?

Reply:
Good catch. @aquarapid  should we increase this? Currently for tables with row lengths < 250bytes the 1K max row limit will kick in before the max buffer size is reached.

Reply:
OK, pushing it to 5k.

Reply:
@rohit-nayak-ps is that a good number?

Reply:
yes, should be a good default